### ISSUE TITLE
DXE-2642 feature request, allow specify property version to data source "akamai_property_hostnames"

### ISSUE THREAD

**Issue by Razacontrib** [akamai/terraform-provider-akamai#413]
Hi there,

Please add possibility to specify property version to data source `akamai_property_hostnames`
like for `akamai_property_rules`

Currently version for `akamai_property_hostnames` is always latest, without possibility to specify version currently activated on staging or production network.

### Terraform Version
Terraform v1.4.6

### Affected Resource(s)

- akamai_property_hostnames

### Expected Behavior
```hcl
data "akamai_property_hostnames" "list" {
  contract_id = "ctr_xxx"
  group_id    = "grp_xxx"
  property_id = "prp_xxx"
  version     = "1"
}
```

### Actual Behavior
hostname list for property  is always from latest version

**Comment by majakubiec** [comment]
Hi @Razacontrib,

Thank you for suggestion; it sounds like a good idea and should be easy to implement.

**Comment by lkowalsk-akamai-com** [comment]
This issue should be fixed now with Akamai Terraform Provider 5.5.0 release
